rust/trace/src/chrome_trace_dump.rs

```diff
@@ -82,6 +82,42 @@ where
     serde_json::from_reader(input).map_err(Error::Json)
 }
 
+/// Save tracing data to path pointed to by the environment variable TRACE_OUTPUT, using the Trace
+/// Viewer format. Save path defaults to `./target/trace_output.trace`. Trace file can be opened
+/// with the Chrome browser by visiting the URL `about:tracing`. If `sorted_chronologically` is true
+/// then sort output traces chronologically by time of creation.
+pub fn save_traces(sorted_chronologically: bool) {
+    use std::env;
+
+    let traces = if sorted_chronologically {
+        super::samples_cloned_sorted()
+    } else {
+        super::samples_cloned_unsorted()
+    };
+
+    let trace_output_path = match env::var("TRACE_OUTPUT") {
+        Ok(output_path) => output_path,
+        Err(_) => {
+            println!("Environment variable TRACE_OUTPUT not set, defaulting to ./target/trace_output.trace");
```

Comment:
all of these `println` should probably be `eprintln`... don't think it's likely to be a problem in this case, but as xi uses stdout to communicate if this was used there it would cause a crash.